File: SAP_APIv1/SAP_API/DTOs/Responses/StartPreparing/StartPreparingResponse.cs
using System;
using System.Collections.Generic;

namespace SAP_API.DTOs.Responses
{
    public class StartPreparingResponse
    {
        public Guid Id { get; set; }
        public string Code { get; set; }
        public int BakingTimeInMins { get; set; }
        public int BakingTempInC { get; set; }
        public DateTime BakingProgrammedAt { get; set; }
        public Guid OvenId { get; set; }
        public string OvenCode { get; set; }
        public List<StartPreparingFromLocationResponse> Locations { get; set; }

    }
}

File: bakery-app/src/models/Responses/StartPreparing/StartPreparingResponse.ts
import LocationToPrepareFromResponse from "./LocationToPrepareFromResponse";

export default interface StartPreparingResponse {
    id: string;
    code: string;
    bakingTimeInMins: number;
    bakingTempInC: number;
    bakingProgrammedAt: Date
    ovenId: string;
    ovenCode: string;
    locations: LocationToPrepareFromResponse[];
}
